content/en/seachat/seachat-manual/02-create-agent/04-seachat-conversations.md

```diff
@@ -44,3 +44,34 @@ In each conversation row, there will be an icon indicating the conversation chan
 </center>
 
 For conversations with phone icons, you can download the audio conversation by clicking on the download icon inside the conversation. The audio conversation will be downloaded in `mp3` format. This is a useful tool for quality assurance and training purposes, as you can listen to the conversation between the agent and the user and provide critical feedback to fine-tune the agent's responses.
+
+## Download Agent Conversation History
+
+If you wish to download the conversation history of certain agent conversations, you can do so by navigating to **Workspace** -> **Agents**. 
+
+Click on **Download Conversations** and Choose the year of the conversation history you wish to download. 
+
+<br/>
+<center>
+<a style="border-radius: 0.4rem; cursor: zoom-in;" href="/images/seachat/en/seachat-conversation/download-conversation-history.png" target="_blank">
+<img width="80%" style="border-radius: 0.4rem" src="/images/seachat/en/seachat-conversation/download-conversation-history.png" alt="">
+</a>
+
+**Download Conversation History**
+</center>
+
+This process might take a while depending on the size of the conversation history. Once the download is ready, you will receive a system notification says `AI agent conversation history export successful. Please click link or copy the link to browser to download. The link will expire in 24 hours.`
+
+Click on the link to download the conversation history in a zip file format. 
+
+You will see the conversation history in a CSV file format. The CSV file contains the following columns:
+
+| Sender type | Channel type | Sender name | Time in GMT | Message | Data |
+|-------------|--------------|-------------|-------------|---------|------|
+
+- **Sender type**: The type of sender. You can identify whether the sender is the agent, system notification, user, etc.
```

Comment:
@hellokellyworld Hi Kelly, can you help me confirm this part of the wiki? I want to explain to the users what these fields mean and what data they can expect. I did a simple generalization of these fields here already. Your confirmation will help a lot thanks!

Reply:
Hi Jun,
Overall, it provides the essential information for users to understand these column headers. I believe we can offer more complete explanations if we can provide some possible values for certain columns.

For example, `Channel type` will definitely be one of the following:
```
* System (this means system message)
* WebChat
* SeaAuth (this is usually for verification purpose)
* Line 
* WhatsApp
* Messenger
* Instagram
* Voice (this means phone calls)
* ThirdPartyClient
* AgentPhone (this means phone calls from SeaChat voice agent, usually for token verification purposes)
* SeaX_Call (this means phone calls through SeaX)
* SeaX_SMS (this means SMS through SeaX)
```
For `Message`, it is usually text most of the time. However, messages starting with a slash / indicate a command, meaning the user selected a menu button in the UI that triggers a system command. For example:
```
* /live_agent: User requests conversation transfer to a live agent.
* /submit_form: User submits a form. If an entity is involved, the command will appear as /submit_form{"entity_name":"entity_value"}.
* /clear_history: User requests the bot to forget the conversation history and start fresh.
* /ai_agent: User requests conversation transfer to an AI agent.
```

For `Data`, it can contains the following :
* Image URL that gets sent in the message
* Recording file URL of the voice call
* A form that the user completes during the chat